nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/AbstractMongoProcessor.java

```diff
@@ -52,6 +62,32 @@
         .required(true)
         .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
         .build();
+    public static final PropertyDescriptor SSL_CONTEXT_SERVICE = new PropertyDescriptor.Builder()
+        .name("SSL Context Service")
+        .description("The SSL Context Service used to provide client certificate information for TLS/SSL "
+                + "connections.")
+        .required(false)
+        .identifiesControllerService(SSLContextService.class)
+        .build();
+    public static final PropertyDescriptor CLIENT_AUTH = new PropertyDescriptor.Builder()
+        .name("Client Auth")
```

Comment:
`.name` should be a machine-safe string (i.e. `ssl-client-auth`) which will remain constant over the life of the processor because it is used for object resolution when loading from the `flow.tar.gz` file. For the human-readable value to display in the UI, please use `.displayName`. This can be changed without affecting object resolution (for future renaming, internationalization, etc.).